src/basho_bench_duration.erl

```diff
@@ -67,8 +67,8 @@ handle_call(remaining, _From, State) ->
     maybe_end({reply, Remaining, State}).
 
 
-handle_cast({worker_stopping, WorkerPid}, State) -> 
-    case basho_bench_worker_sup:active_workers() -- [WorkerPid] of 
+handle_cast({worker_stopping, WorkerId}, State) ->
+    case basho_bench_worker_sup:active_workers() -- [WorkerId] of
```

Comment:
Good catch. Let's switch the name here back to `WorkerPid` though, as the value passed in is still a pid, not an id.